atd-vze/src/views/Crashes/Recommendations.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import { Card, CardHeader, CardBody, CardFooter, Table, Button } from "reactstrap";
+import { recommendationsDataMap } from "./recommendationsDataMap";
+import { GET_RECOMMENDATIONS } from "../../queries/recommendations";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare fatality review board recommendations component
+const Recommendations = ({ crashId }) => {
+
+  // fetch data from database using graphQL query
+  const { loading, error, data, refetch } = useQuery(GET_RECOMMENDATIONS, {
+    variables: { crashId },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  // if crash has data in recommendations table, display it
+  const displayData = (field) => {
+    if (data.recommendations[0]) {
+        if (field === "partner") {
+            return data.recommendations[0].atd__coordination_partners_lkp.description
+        } else if (field === "status") {
+            return data.recommendations[0].atd__recommendation_status_lkp.description
+        } else if (field === "note") {
+            return data.recommendations[0].text
+        }}
+  }
+
+  const fieldConfig = recommendationsDataMap[0];
+
+    return(
+        <Card>
+            <CardHeader>{fieldConfig.title}</CardHeader>
+            <CardBody>
+                <Table>
+                    <tr style={{width: "100%"}}>
+                        <td style={{width: "35%"}}>
+                            <b>{fieldConfig.fields.coordination_partner_id.label}</b>
+                            {displayData("partner")}
+                        </td>
```

Comment:
i like that you're driving this with a config, but fwiw with just three cells i'd be fine hardcoding it (and it may be easier to grok)

```javascript
          <td style={{width: "35%"}}>
              <b>Fatality Review Board Recommendations</b>
               {data.recommendations[0].atd__coordination_partners_lkp.description}
          </td>
```

Reply:
yeah i was unsure what would be best practice. at this point it feels like a datamap isnt necessary for this component, am i wrong?

Reply:
👍 either way is fine